Build.cmd

```diff
@@ -1,2 +1,6 @@
 @echo off
-powershell -noprofile -executionPolicy RemoteSigned -file "%~dp0\build\scripts\build.ps1" -build %*
+if "%*" == "" (
+    powershell -noprofile -executionPolicy RemoteSigned -file "%~dp0\build\scripts\build.ps1" -build
```

Comment:
nit: Might be useful to explicitly specify `-Version x.y` for whichever version we depend on. It will guarantee we don't accidentally take a dependency on functionality only available in newer PowerShell versions.